src/app/page.tsx

```diff
@@ -1,12 +1,14 @@
 import Image from "next/image";
 import styles from "./page.module.css";
+import "./nav.module.css";
+import Navbar from "./nav";
 
 export default function Home() {
   return (
     <main className={styles.main}>
-      <div className={styles.description}>
-        <p>Homepage</p>
-      </div>
+      <div className={styles.description}></div>
+
+      <Navbar></Navbar>
```

Comment:
When components do not have any children, you can use them as a self-closing tag like this:
`<Navbar />`. That way, you don't need to type its name twice.